maths/binary_multiplication.py

```diff
@@ -24,6 +50,35 @@ def b_expo(a: int, b: int) -> int:
 
 
 def b_expo_mod(a: int, b: int, c: int) -> int:
+    """
+    Calculate the result of (a * b) % c using binary exponentiation and modular arithmetic.
```

Comment:
Please fix the ruff error on this line. The line is too long (88 chars max)

Reply:
Hi, I have fixed this Can you please check it now.